ext/datasource/k8s/deploy/crds/datasource.sentinel.com_flowrules_crd.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: flowrules.datasource.sentinel.com
+  namespace: default
+  labels:
+    sentinel-datasource: k8s
+spec:
+  group: datasource.sentinel.com
+  names:
+    kind: FlowRules
+    listKind: FlowRulesList
+    plural: flowrules
+    singular: flowrule
+    shortNames:
+    - frs
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      description: FlowRules is the Schema for the flowrules API
+      properties:
+        apiVersion:
+          description: 'Kind is a string value representing the REST resource this object represents.'
+          type: string
+        kind:
+          description: 'APIVersion defines the versioned schema of this representation of an object.'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: FlowRulesSpec defines the desired state of FlowRules
+          properties:
+            rules:
+              description: 'rules is the flow rule list'
+              items:
+                properties:
+                  resource:
+                    type: string
+                    minLength: 1
+                    maxLength: 100
+                  limitOrigin:
+                    type: string
+                    minLength: 1
+                    maxLength: 100
+                  metricType:
+                    type: string
+                    enum:
+                    - Concurrency
+                    - QPS
+                  count:
+                    type: number
+                    format: float
+                  relationStrategy:
+                    type: string
+                    enum:
+                    - Direct
+                    - AssociatedResource
+                  controlBehavior:
+                    type: string
+                    enum:
+                    - Reject
+                    - WarmUp
+                    - Throttling
+                    - WarmUpThrottling
+                  refResource:
+                    type: string
+                    minLength: 1
+                    maxLength: 100
+                  warmUpPeriodSec:
+                    format: int32
+                    type: integer
+                    minimum: 0
+                  maxQueueingTimeMs:
+                    format: int32
+                    type: integer
+                    minimum: 0
+                required:
+                - resource
+                - limitOrigin
```

Comment:
Only resource, metricType and count are required. For other attributes, we could provide default values.

Reply:
make sense